pkg/daemon/daemon.go

```diff
@@ -120,6 +151,12 @@ func New(
 // responsible for triggering callbacks to handle updates. Successful
 // updates shouldn't return, and should just reboot the node.
 func (dn *Daemon) Run(stop <-chan struct{}) error {
+	// Catch quickly if we've been asked to run once.
+	if dn.onceFrom != "" {
+		glog.V(2).Info("Running once per request")
```

Comment:
Is this logging clear enough? Would "daemon running once per request" be clearer? Thoughts?

Reply:
I like your wording better. How about I rework some of the log strings in a follow up?

Reply:
Sounds great!